provision-contest/ansible/roles/dj_notify/files/dj_notify.py

```diff
@@ -0,0 +1,86 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import json
+import subprocess
+import gi
+import os
+import webbrowser
+import subprocess
+import traceback
+gi.require_version('Notify', '0.7')
+from gi.repository import Notify
+
+HOSTNAME = "0.0.0.0"
+PORT = 9999
+ALSA_DEVICE = os.environ['ALSA_DEVICE']
+NOTIFICATION_SOUND = os.environ['NOTIFICATION_SOUND']
+NOTIFICATION_SOUND_VOLUME = int(os.environ['NOTIFICATION_SOUND_VOLUME'])
+
+
+def on_notification_closed(notification):
+    print(f"Notification {notification.id} closed.")
+
+
```

Comment:
Does the linter require the double whitelines?

Reply:
Yes, it did